task-sdk/src/airflow/sdk/api/datamodels/_generated.py

```diff
@@ -181,10 +181,10 @@ class TIDeferredStatePayload(BaseModel):
     )
     state: Annotated[Literal["deferred"] | None, Field(title="State")] = "deferred"
     classpath: Annotated[str, Field(title="Classpath")]
-    trigger_kwargs: Annotated[dict[str, Any] | str | None, Field(title="Trigger Kwargs")] = None
+    trigger_kwargs: Annotated[JsonValue | str | None, Field(title="Trigger Kwargs")] = None
     trigger_timeout: Annotated[timedelta | None, Field(title="Trigger Timeout")] = None
     next_method: Annotated[str, Field(title="Next Method")]
-    next_kwargs: Annotated[dict[str, Any] | None, Field(title="Next Kwargs")] = None
+    next_kwargs: JsonValue | None = None
```

Comment:
`JsonValue` is `dict | list | str | int | float | bool | None` in Pydantic. But `next_kwargs` is semantically always a dict - it represents keyword arguments that get unpacked as `**next_kwargs` in `execute_callable(context, **next_kwargs)`. Changing the type to `JsonValue` is incorrect because:

serde output is still a dict: `serde.serialize({"foo": datetime})` returns `{"foo": {"__classname__": ...}}` - the outer container is still a dict

Reply:
Right, outside the context in https://github.com/apache/airflow/pull/59711/changes#diff-5bef10ab2956abf7360dbf9b509b6e1113407874d24abcc1b276475051f13abfR1387-R1399, `**next_kwargs` can be done. Good call.

In any case, now `next_kwargs` is `dict[str, JsonValue]`, so that part should be OK